client/js/components/statement/publicStatementModal/StatementModal.vue

```diff
@@ -26,14 +26,15 @@
       </template>
       <template v-slot:closeButton>
         <button
-          type="button"
-          @click="toggleModal(false)"
-          aria-label="Stellungnahme-Formular minimieren"
           aria-describedby="statementDialogCloseTitle"
-          :class="prefixClass('c-statement__close btn-icns u-m-0 u-p-0')">
+          aria-label="Stellungnahme-Formular minimieren"
+          :class="prefixClass('c-statement__close btn-icns color-highlight u-m-0_25 p-0 absolute u-right-0')"
+          title="Stellungnahme-Formular minimieren"
```

Comment:
this should be translated (and aria-label)

Reply:
Unfortunately I can't open or edit the core messages+intl file

Reply:
Used trans key in ee0bcff032939e6551b194646a72efa79cc6e604